src/Animals/CompetingAnimal.py

```diff
@@ -0,0 +1,16 @@
+class CompetingAnimal:
+    def __init__(self, name: str, start_point: int, step_per_sec: int, step_sign: str):
+        self.name = name
+        self.start_point = start_point
+        self.step_per_sec = step_per_sec
+        self.step_sign = step_sign
+        self.steps = []
+        for i in range(start_point):
+            self.steps += 'X'
```

Comment:
`self.steps` is getting initialized, there's a way to do it in as a one-liner :)